src/main/scala/net/cakesolutions/CakePlatformPlugin.scala

```diff
@@ -1,50 +1,57 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
-package net.cakesolutions
 // Copyright 2015 - 2016 Sam Halliday (derived from sbt-sensible)
```

Comment:
this is wrong and will conflict with `sbt-header`. You are not legally allowed to remove the `sbt-sensible` copyright header as this is a derived works.

scalastyle is rubbish, I don't know why you want to use it.